src/openrct2/ride/CableLift.cpp

```diff
@@ -242,17 +240,13 @@ static bool sub_6DF01A_loop(Vehicle* vehicle)
         {
             _vehicleVAngleEndF64E36 = TrackDefinitions[trackType].vangle_end;
             _vehicleBankEndF64E37 = TrackDefinitions[trackType].bank_end;
-            TileElement* trackElement = map_get_track_element_at_of_type_seq(
-                { vehicle->track_x, vehicle->track_y, vehicle->track_z }, trackType, 0);
+            TileElement* trackElement = map_get_track_element_at_of_type_seq(vehicle->TrackLocation, trackType, 0);
 
-            CoordsXYE input;
             CoordsXYE output;
             int32_t outputZ;
             int32_t outputDirection;
 
-            input.x = vehicle->track_x;
-            input.y = vehicle->track_y;
-            input.element = trackElement;
+            CoordsXYE input = { vehicle->TrackLocation, trackElement };
```

Comment:
not a huge deal, but we've been mostly doing this, in these cases:
```suggestion
            auto input = CoordsXYE{ vehicle->TrackLocation, trackElement };
```